### PR TITLE
Feature:lesson3

### PR DESCRIPTION
**Pull request by kanakonagiri**
# [lesson3](リンク)
## [comment]
I referred to the code created in Lesson 2.
Should I write it differently when adding more "li"?

### CODE REVIEW

File: lesson3/index.html [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+    <title>TerraceTechフロントエンドエンジニア養成所</title>
+  </head>
+  <body>
+    <div>
+      <ul id="js-lists"></ul>
+    </div>
+    <script src="script.js"></script>
```
Comment (kenmori) [inline lesson3/index.html]: I prefer to `index.js`

Comment (kanakonagiri) [reply lesson3/index.html]: I don't understand why `index.js` is preferred.

**Comment by kanakonagiri** [comment]
> use array to implement in this lesson.
I rewrote using an array.
d43c46d

**Comment by kenmori** [comment]
approve

**Pull request merged**